packages/serverless/src/apollo/app/types/permission.ts

```diff
@@ -0,0 +1 @@
+export type TPermissionGroup = 'owner' | 'group' | 'other';
```

Comment:
You can reuse `PermissionType` in `@zkdb/permission`

Reply:
Deleted it. It not use anywhere